daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ExplicitDisclosureTest.scala

```diff
@@ -8,11 +8,7 @@ import com.daml.lf.command.ContractMetadata
 import com.daml.lf.crypto.Hash
 import com.daml.lf.data.Ref.{IdString, Party}
 import com.daml.lf.data.{FrontStack, ImmArray, Ref, Struct, Time}
-import com.daml.lf.interpretation.Error.{
```

Comment:
All "disclosure preprocessing" testing has now been moved into the `PreprocessorSpec.scala` file.